File: client/src/app/_models/params/userHistoryParams.ts
export class UserHistoryParams
{
    id = 0;
    userHistoryHeaderId= 0;
    personType = '';
    personName = '';
    personId? = 0;
    customerOfficialId = 0;
    applicationNo = 0;
    aadharNo = '';
    emailId = '';
    mobileNo = '';
    dateAdded: Date=new Date();
    categoryRef = '';
    status = '';
    concluded = null;        //default
    createNewIfNull=false;
    userName =  '';
    search = '';
    sort = 'applicationNo';
    pageNumber=1;
    pageSize=10;
}

File: api/Params/Admin/UserHistoryParams.cs
namespace api.Params.Admin
{
    public class UserHistoryParams: PaginationParams
    {
        public int UserHistoryId {get; set;}
        public int CandidateId { get; set; }
        public string ResumeId { get; set; }
        public string CandidateName { get; set; }
        public int ApplicationNo {get; set;}
        public string CategoryRef { get; set; }
        public string Status { get; set; }
        public bool? Concluded {get; set;}
        public string UserName {get; set;}
    }
}